### ISSUE TITLE
Conditions with blocks return differently when the looped stuff isn't a variable

### ISSUE THREAD

**Issue by TPGamesNL** [SkriptLang/Skript#2175]
### Description
The solid condition doesn't work in loops when the given values to loop aren't variables (this is a terrible description but you should look at Steps to Reproduce, it's more understandable).
Same for: `is transparent`, `is flammable`

### Steps to Reproduce
```
on rightclick on sand with diamond hoe:
	loop block above and block below:
		message "&21 &r%loop-value% (%""&atrue"" if loop-value is solid, else (""&cfalse"")%&r)"
	set {_b::*} to block above and block below
	loop {_b::*}:
		message "&32 &r%loop-value% (%""&atrue"" if loop-value is solid, else (""&cfalse"")%&r)"
```
![image](https://user-images.githubusercontent.com/29547183/59706911-cdfc5700-9201-11e9-9346-acda0e77cf54.png)

### Expected Behavior
The condition should return `true` or `false` consistently in both loops.

### Server Information
* **Server version/platform:** Paper-94 (1.14.2)
* **Skript version:** 2.4-alpha3

**Comment by TheBentoBox** [comment]
Well that's weird.

**Comment by Wealthyturtle** [comment]
Will try and investigate this later

**Comment by Wealthyturtle** [comment]
Oddly enough, I found that this was caused due to the internal use of ItemStack, hence this issue is fixed by #2510
